Packages/MIES/MIES_Constants.ipf

```diff
@@ -1223,6 +1223,15 @@ Constant WINTYPE_CAMERA = 15
 Constant WINTYPE_GIZMO = 17
 /// @}
 
+/// @name objectType codes of the Igor GetIndexedObjectName and GetIndexedObjectNameDFR function
+/// @anchor GetIndexedObjectName_ObjectTypes
+/// @{
+Constant OBJTYPE_WAVES = 1
+Constant OBJTYPE_NUMVARS = 2
+Constant OBJTYPE_STRVARS = 3
+Constant OBJTYPE_DATAFOLDERS = 4
+/// @}
```

Comment:
We have these already as `COUNTOBJECTS_WAVES` and friends.

Reply:
ah ok